internal/server/filesystem/internal/v1alpha1/conversion_generated.go

```diff
@@ -0,0 +1,34 @@
+// Code generated by csi-proxy-gen. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	"github.com/kubernetes-csi/csi-proxy/client/api"
+	pb "github.com/kubernetes-csi/csi-proxy/client/api/filesystem/v1alpha1"
+	"github.com/kubernetes-csi/csi-proxy/internal/server/filesystem/internal"
+)
+
+func autoconvert_pb_ComputeDoubleRequest_To_internal_ComputeDoubleRequest(in *pb.PathExistsRequest, out *internal.PathExistsRequest) error {
+	out.Path = in.Path
+	return nil
+}
+
+// convert_pb_PathExistsRequest_To_internal_PathExistsRequest is an autogenerated conversion function.
+func convert_pb_PathExistsRequest_To_internal_PathExistsRequest(in *pb.PathExistsRequest, out *internal.PathExistsRequest) error {
+	return autoconvert_pb_ComputeDoubleRequest_To_internal_ComputeDoubleRequest(in, out)
+}
+
+func autoconvert_internal_ComputeDoubleResponse_To_pb_ComputeDoubleResponse(in *internal.PathExistsResponse, out *pb.PathExistsResponse) error {
```

Comment:
Is this expected? The ComputeDouble stuff is from the dummy API group. They should not show up under the filesystem API group, correct?

Reply:
Same :)